sdk/storage/azure_storage_blob/src/storage_utils.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+pub struct StorageUtils;
+
+impl StorageUtils {
```

Comment:
This isn't C#. ;) You can declare just functions. `azure_core` has quite a few, as do most of the crates we call. I would just define this as a standalone function. There's no benefit of putting it in a type that otherwise has no purpose.